### PR TITLE
[MOBILE-1297] Update CHANGELOG for .NET 12.0 & iOS 13.1 release.

### PR DESCRIPTION
**Pull request by BrianBatchelder**
### What do these changes do?
Prepare Xamarin repo for .NET 12.0 & iOS 13.1 release

### Why are these changes necessary?
Regular Xamarin update plus some bug fixes.

### How did you verify these changes?
Lots of testing was done on the PRs that feed into this release. Local build succeeded. Local sample app tests revealed one problem which is fixed in the second commit.

### Anything else a reviewer should know?
Version numbers were updated by previous commits.

I had to remove the SampleContentExtension from iOS sample app:
- During the iOS SDK 13.0.4 I created NotificationContentExtension bindings for the new NotificationContentExtension module.
- I also created a SampleContentExtension project for the iOS sample to match our SampleServiceExtension thinking that would add some default content extension functionality.
- Unfortunately, what adding the empty SampleContentExtension project did was break media push notifications. 
- I will create a ticket to explore adding content extension functionality to our Xamarin sample app.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,39 @@ This library provides official bindings to the Airship SDK, as well as sample ap
...
+- nupkg installations failing to link with missing symbol errors.
+- .NETStandard and PCL app crashes when adding a custom event (`Airship.Instance.AddCustomEvent();`)
+Apps experiencing either of these issues should update to this version.
+
+Changes
+-------
+- Updated iOS SDK to 13.1.0
+- Created new package layout for the new iOS SDK modules:
+  - New packages
+    - urbanairship.ios.automation
```
Comment (rlepinski) [inline CHANGELOG.md]: is everything lower cased ?

Comment (BrianBatchelder) [reply CHANGELOG.md]: They are on [NuGet](https://www.nuget.org/packages?q=urbanairship).

Comment (BrianBatchelder) [reply CHANGELOG.md]: Maybe line 24 should be `Created new nuget package layout for the new iOS SDK modules:`

**Comment by BrianBatchelder** [comment]
The sample content extension affected the sample service extension. I’m guessing it override some iOS default functionality without implementing its own default functionality. 

I created a ticket: MOBILE-1310

**Pull request merged**